e2e/Makefile

```diff
@@ -29,7 +29,7 @@ PATH := $(GOPATH)/bin:$(PATH)
 default: tools
 
 clean:
-	rm -fr manifests
+	rm -rf manifests
```

Comment:
I want to complain, if I'm not mistaken this is just a noise, and I would like us to strive for clean PRs.